.github/workflows/eas_build.yaml

```diff
@@ -0,0 +1,29 @@
+on:
+  push:
+    branches:
+      - main
```

Comment:
**discussion**: Do we want to have these builds with each `main` merge. Or maybe we should build when label is added to PR? 🤔